app/src/main/java/org/simple/clinic/recentpatientsview/LatestRecentPatientsEffectHandler.kt

```diff
@@ -19,6 +26,23 @@ class LatestRecentPatientsEffectHandler @AssistedInject constructor(
   fun build(): ObservableTransformer<LatestRecentPatientsEffect, LatestRecentPatientsEvent> {
     return RxMobius
         .subtypeEffectHandler<LatestRecentPatientsEffect, LatestRecentPatientsEvent>()
+        .addTransformer(LoadRecentPatients::class.java, loadRecentPatients())
         .build()
   }
+
+  private fun loadRecentPatients(): ObservableTransformer<LoadRecentPatients, LatestRecentPatientsEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .switchMap { effect ->
+            userSession
+                .loggedInUser()
+                .subscribeOn(schedulers.io())
+                .filterAndUnwrapJust()
+                .take(1)
+                .switchMap(facilityRepository::currentFacility)
```

Comment:
Aren't we using `private val currentFacility: Lazy<Facility>` to get current facility or will that be done in later commits?

Reply:
We will not change this during strangulation and retain whatever method we were using in the controller. We'll switch to using the injected providers in the final clean up PRs.